lib/sentry.ex

```diff
@@ -155,17 +159,40 @@ defmodule Logger.Backends.Sentry do
     end
   else
     defp log_event(:error, metadata, msg, state) do
-      Sentry.capture_exception(
-        generate_output(:error, metadata, msg),
+      log_exception(metadata, msg)
+      state
+    end
+
+    defp log_event(level, metadata, msg, state) do
+      Sentry.capture_message(
+        generate_output(level, metadata, msg),
         generate_opts(metadata, msg)
```

Comment:
Here maybe get keyword which contain value is `nil`.

Reject value is `nil` only after `generate_opts`  rather than others (like: https://github.com/Tubitv/logger_sentry/pull/10/files#diff-05162b4f01eb6be751a3063e1488e5daR189, https://github.com/Tubitv/logger_sentry/pull/10/files#diff-05162b4f01eb6be751a3063e1488e5daR217 [actually need here]) maybe better.

Reply:
I meant to exclude nil values on the very last but your suggestion is okay.